client/components/contributepage/ResourcesSection.js

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import { breakpoint } from "../../frontend-config";
+
+const Wrapper = styled.div`
+  background: ${props => props.theme.colors.grey};
+  padding: 45px 25px 65px;
+
+  @media (min-width: ${breakpoint("lg")}) {
+    padding: 115px 25px 190px;
+  }
+`;
+
+const Title = styled.h2`
+  font-size: 39px;
+  font-weight: bold;
+  color: ${props => props.theme.colors.light};
+`;
+
+const ContentWrapper = styled.div`
+  max-width: 1000px;
+  margin: 0 auto;
+`;
+
+const ButtonsWrapper = styled.div`
```

Comment:
These stack a bit odd on mobile, maybe have this be flex-column when under the 'md' breakpoint.

Reply:
Will have a play with this, thanks :+1:

Reply:
The resources at the bottom look great! Thanks for taking the time to get them to look nice on mobile.